app/controllers/spree/users_controller_decorator.rb

```diff
@@ -1,3 +1,20 @@
 Spree::UsersController.class_eval do
   layout 'darkswarm'
+  before_filter :set_credit_card, only: :show
```

Comment:
See my suggestion in enterprises_controller.rb

Reply:
Resolved